README.md

```diff
@@ -29,3 +30,7 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+My asymptotic runtime in terms of the number of comparisions is $\Theta(n*n!)$ because it makes a comparision for every item in the list where any item in the list could be the biggest. 
```

Comment:
Where does the $n!$ come from?

Reply:
Last night I was thinking about the permutations the list could be in and I was way off course on how complexity works. It's $\Theta(n^2)$.